run.py

```diff
@@ -74,10 +74,11 @@
 if host == '.':
     path = domain
 
-print("cf files letsencrypt app/conf/live/" + path + "/cert.pem")
-print("cf files letsencrypt app/conf/live/" + path + "/chain.pem")
-print("cf files letsencrypt app/conf/live/" + path + "/fullchain.pem")
-print("cf files letsencrypt app/conf/live/" + path + "/privkey.pem")
+print("cf ssh letsencrypt -c 'cat ~/app/conf/live/" + path + "/cert.pem' > cert.pem")
```

Comment:
Can you drop the files/ssh change for now please?
We need to plan for how to handle both Diego and DEA.